### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'definitions')

### ISSUE THREAD

**Issue by muellerj93** [sapmentors/cds-pg#385]
Hi,

I recently updated to `cds-pg` v0.1.35.
When deploying my project, I'm running into the following error:

`TypeError: Cannot read properties of undefined (reading 'definitions')`

with the callstack:

```
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR cds-dbm deploy [services]
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR Dynamically identifies changes in your cds data model and deploys them to the
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR database
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR Options:
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR --help           Show help                                       [boolean]
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR --version        Show version number                             [boolean]
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR -s, --service                                                [default: ["db"]]
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR -a, --auto-undeploy
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR -d, --dry
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR -l, --load-via
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR -c, --create-db
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR TypeError: Cannot read properties of undefined (reading 'definitions')
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at PGSelectBuilder._columns (/home/vcap/deps/1/node_modules/cds-pg/lib/pg/sql-builder/SelectBuilder.js:84:32)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at PGSelectBuilder.build (/home/vcap/deps/1/node_modules/@sap/cds/libx/_runtime/db/sql-builder/SelectBuilder.js:81:10)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at build (/home/vcap/deps/1/node_modules/@sap/cds/libx/_runtime/db/sql-builder/sqlFactory.js:65:43)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at build (/home/vcap/deps/1/node_modules/@sap/cds/libx/_runtime/db/sql-builder/sqlFactory.js:74:12)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at _cqnToSQL (/home/vcap/deps/1/node_modules/cds-pg/lib/pg/execute.js:148:5)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at executeSelectCQN (/home/vcap/deps/1/node_modules/cds-pg/lib/pg/execute.js:51:34)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at PostgresDatabase._read (/home/vcap/deps/1/node_modules/@sap/cds/libx/_runtime/db/query/read.js:73:10)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at module.exports [as _READ] (/home/vcap/deps/1/node_modules/@sap/cds/libx/_runtime/db/generic/read.js:15:15)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at next (/home/vcap/deps/1/node_modules/@sap/cds/lib/srv/srv-dispatch.js:75:36)
2023-02-27T10:32:31.56+0100 [APP/TASK/deploy_to_postgresql/0] ERR at PostgresDatabase.handle (/home/vcap/deps/1/node_modules/@sap/cds/lib/srv/srv-dispatch.js:79:6)
```

I'm using `@sap/cds` v6.5.0, `cds-dbm` v0.1.2 and `cds-pg` v0.1.35

Is there something wrong with my cds model or is it a bug?

Best regards & Thanks

**Comment by muellerj93** [comment]
Thanks for the hint, but unfortunately my cds config already has this value set:

![image](https://user-images.githubusercontent.com/102537323/222424393-c29c1ba3-60cd-491e-bcc4-a9e0ab50f8a6.png)

**Comment by mikezaschka** [comment]
I published `cds-dbm@0.1.3` to support the latest version of `@sap/cds`, which resolves this issue.
I also update the simple example project (https://github.com/mikezaschka/cap-devtoberfest) to run with all the latest versions of the libraries.
